packages/server/src/config.ts

```diff
@@ -264,6 +265,7 @@ function addDefaults(config: MedplumServerConfig): MedplumServerConfig {
   config.botLambdaLayerName = config.botLambdaLayerName || 'medplum-bot-layer';
   config.bcryptHashSalt = config.bcryptHashSalt || 10;
   config.bullmq = { removeOnComplete: { count: 1 }, removeOnFail: { count: 1 }, ...config.bullmq };
+  config.shutdownTimeoutMilliseconds = config.shutdownTimeoutMilliseconds ?? 30000;
```

Comment:
This is redundant in its current state as the default timeout for `http-graceful-shutdown` is already 30 seconds https://github.com/sebhildebrandt/http-graceful-shutdown#options

I can remove, unless we'd like to adjust the default.

Reply:
30 seconds is appropriate.  I don't have strong feelings about whether to be explicit or implicit here.